### ISSUE TITLE
Winpath: Found this issue of broken pipe.

### ISSUE THREAD

**Issue by Abhinickz** [Abhinickz/learn_rust#2]
**Winpath Issue:**
```
[09:06:20 abhinickz@wsl -> work$ find ./ | winpath_rust | head -0
thread 'main' panicked at 'failed printing to stdout: Broken pipe (os error 32)', libstd/io/stdio.rs:692:9
note: Run with `RUST_BACKTRACE=1` for a backtrace.

```

**Full Error:**

```
[09:12:06 abhinickz@wsl -> work$ find ./ | RUST_BACKTRACE=1 winpath_rust | head -0
thread 'main' panicked at 'failed printing to stdout: Broken pipe (os error 32)', libstd/io/stdio.rs:692:9
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
             at libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1: std::sys_common::backtrace::print
             at libstd/sys_common/backtrace.rs:71
             at libstd/sys_common/backtrace.rs:59
   2: std::panicking::default_hook::{{closure}}
             at libstd/panicking.rs:211
   3: std::panicking::default_hook
             at libstd/panicking.rs:227
   4: std::panicking::rust_panic_with_hook
             at libstd/panicking.rs:463
   5: std::panicking::begin_panic_fmt
             at libstd/panicking.rs:350
   6: std::io::stdio::_print
             at libstd/io/stdio.rs:692
             at libstd/io/stdio.rs:701
   7: winpath::main
   8: std::rt::lang_start::{{closure}}
   9: std::panicking::try::do_call
             at libstd/rt.rs:59
             at libstd/panicking.rs:310
  10: __rust_maybe_catch_panic
             at libpanic_unwind/lib.rs:105
  11: std::rt::lang_start_internal
             at libstd/panicking.rs:289
             at libstd/panic.rs:374
             at libstd/rt.rs:58
  12: main
  13: __libc_start_main
  14: _start
```

**Comment by Abhinickz** [comment]
**Fixed the issue with commit:819d838eb7e922697812c2a51ff0766d44090e0b** 
#15

**Comment by Abhinickz** [comment]
```
[22:16:52 abhinickz@wsl -> work$ find ./ | winpath_rust | head -0
[22:16:56 abhinickz@wsl -> work$ 

```
